windows/jitsi-meet/index.html

```diff
@@ -0,0 +1,53 @@
+ <!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            body, html {
+                margin: 0;
+                height: 100%;
+                overflow: hidden;
+            }
+            body {
+                -webkit-app-region: drag
```

Comment:
we won't need this if the titlebar is not hidden. in addition, doing this breaks buttons a bit.